core/services/head_tracker.go

```diff
@@ -13,6 +13,15 @@ import (
 	"chainlink/core/utils"
 
 	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	headsReceived = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Kind of a quibble, but using a prefix like `num` or `metric` might make this a little clearer to someone less familiar with this part of the codebase.